releasenotes/notes/show_idle_and_show_barrier-6e77e1f9d6f55599.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    The parameters ``show_idle`` and ``show_barrier`` in the timeline drawers had been replaced by ``idle_wires`` and ``plot_barriers``
```

Comment:
Should we reword the reno to say that the new parameter names have been introduced and the old names will be deprecated in a following release?

Reply:
In my opinion, we should either remove all mentions of deprecation timings in the release note and trust the user to know/read about our deprecation policy OR we should be accurate about when we deprecate and remove things...

That said, I think it is rather minor issue here and I will approve the changes. :-)